### ISSUE TITLE
Split the repo

### ISSUE THREAD

**Issue by brice-morin** [STAMP-project/camp#71]
I'd recommend splitting the repo in two:
- this repo should only contain the CAMP tool
- a new repo where we migrate all the use cases

Right now, building the image for CAMP takes a lot of time, as the build context is increasing a lot with the data produced by the use cases, which are not used by the tool itself.

@fchauvel what do you think?

**Comment by fchauvel** [comment]
Yes, this is a good idea, because the Docker image that is automatically built on each commit has become super fat (1.5 GB). It includes the `samples` directory, which is very convenient for demo purposes.

**Comment by brice-morin** [comment]
To be migrated tomorrow :-)

**Comment by danzone** [comment]
Dear @fchauvel and @brice-morin, consider to use the [STAMP Courseware repo](https://github.com/STAMP-project/stamp-courseware/): we are arranging here all the relevant STAMP documentation, included also presentations, use cases, etc. Would you prefer to keeping it in different location, is fine as well, of course, and we'll add a reference to your repo.

**Comment by fchauvel** [comment]
I moved all case-studies except the AtoS one. @fermenreq and @jesus-gorronogoitia ask we wait for all experiments to be complete before to proceed.
